<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appoTuple</class>
 <widget class="QWidget" name="appoTuple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="patientName"/>
   </item>
   <item>
    <widget class="QLineEdit" name="patientIdCard"/>
   </item>
   <item>
    <widget class="QLineEdit" name="date"/>
   </item>
   <item>
    <widget class="QLineEdit" name="timeSlot"/>
   </item>
   <item>
    <widget class="QLineEdit" name="status"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>创建病例</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>病例管理</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
